requirements.txt

```diff
@@ -35,3 +35,6 @@ pydispatcher>=2.0.5
 
 # pyyaml
 pyyaml
+
+# psutil
```

Comment:
Make sure the comment reflects the component/platform where it is used. So this would be `sensor.systemmonitor`. Also, make sure you pull in the latest dev changes, I just pushed a minor cleanup to requirements.txt to the dev branch.